### DISCUSSION TITLE
Case insensitive for requested path names

### DISCUSSION THREAD

**Discussion by ardenwick** [khoa-io/git-lfs-server#46] [Q&A] [answered]
Some repo mirrors are mounted in `/r` inside git-lfs-server container, but cloning 'maratyszcza/fp16' fails with:
```
Cloning into '/repo/usearch/fp16'...
fatal: '/r/maratyszcza/fp16' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
Actually the requested repo `maratyszcza/fp16` exists as `/r/Maratyszcza/FP16`, so for the requests to be handled as compliant with github URLs (requesting https://github.com/maratyszcza/fp16  and https://github.com/Maratyszcza/FP16 both hit the same repo), please make it case insensitive.

**Comment by khoa-io** [comment]
I don't recommend storing the repos inside a container. Please use [volumes](https://docs.docker.com/storage/volumes/).
Moreover, this looks like a git's problem: `'/r/maratyszcza/fp16' does not appear to be a git repository`. Please check that.

**Comment by ardenwick** [comment, marked as answer]
Changing all my local bare repository directories (as stored inside `/r`) to lower case and requesting then with all lower case URLs actually solved the `does not appear to be a git repository` problem.
